apis/beacon/blocks/blinded_blocks.yaml

```diff
@@ -0,0 +1,43 @@
+post:
+  tags:
+    - Beacon
+    - ValidatorRequiredApi
+  summary: "Publish a signed block."
+  operationId: "publishBlindedBlock"
+  description: |
+    Instructs the beacon node to use the components of the `SignedBlindedBeaconBlock` to construct and publish a 
+    `SignedBeaconBlock` by swapping out the `transactions_root` for the corresponding full list of `transactions`.
```

Comment:
I think something is not correct here. Which one do we actually publish, a `SignedBeaconBlock` or a `SignedBlindedBeaconBlock`?

Reply:
The beacon node can only publish a `SignedBeaconBlock` via gossip but this endpoint accepts a `SignedBlindedBeaconBlock`. So the beacon node will have to transform the `SignedBlindedBeaconBlock` into a `SignedBeaconBlock` before publishing.

The wording here was meant to cover what using blinded blocks in the normal proposal flow would look like. If flashbots is used instead, this is a little different because `SignedBlindedBeaconBlock`'s are "published" to the flashbots relay, but in reality the flashbots relay will have to make this same `SignedBlindedBeaconBlock` to `SignedBeaconBlock` transformation before *actually* publishing the block via gossip. 

> Also would it make sense to extend https://ethereum.github.io/beacon-APIs/#/Beacon/publishBlock with a new type of block for Bellatrix?

This should be sufficient, right? https://github.com/ethereum/beacon-APIs/blob/67b8f4de30039bf46e92df2e78eebda7461be105/apis/beacon/blocks/blocks.yaml#L23

Reply:
Thanks for explaining! And yes, the linked line is sufficient. I just missed it.